docs/dev/c4/containers.md

```diff
@@ -0,0 +1,37 @@
+```mermaid
+C4Context
+  title [Container] FloPy
+
+  Boundary(mf6, "MODFLOW 6"){
+    System(src, "Source code",)
+    System(dfn, "Definition files",)
+  }
+
+  Boundary(flopy, "FloPy") {
+
+    Boundary(devs, "Developer APIs") {
```

Comment:
Maybe the boundaries are a bit too nested now. User API is now inside of Developer API. It would be nice if we split those up.

Reply:
Will do. At first I figured developer functionality was a superset of user functionality, so it made sense to nest, but the APIs are to be separate- and these boundaries are "systems" not "scopes"